roles/benchmarking_run_nvidiadl_ssd/files/entrypoint.sh

```diff
@@ -1,10 +1,49 @@
 #! /bin/bash
 
+DATASET_DIR=/storage
+if [ ! -f ${DATASET_DIR}/annotations/bbox_only_instances_train2017.json ]; then
+    echo "Prepare instances_train2017.json ..."
+    ./prepare-json.py \
+        "${DATASET_DIR}/annotations/instances_train2017.json" \
+        "${DATASET_DIR}/annotations/bbox_only_instances_train2017.json"
+fi
+
+if [ ! -f ${DATASET_DIR}/annotations/bbox_only_instances_val2017.json ]; then
+    echo "Prepare instances_val2017.json ..."
+    ./prepare-json.py --keep-keys \
+        "${DATASET_DIR}/annotations/instances_val2017.json" \
+        "${DATASET_DIR}/annotations/bbox_only_instances_val2017.json"
+fi
+
+export TORCH_HOME=${DATASET_DIR}/torchvision
+
 ls /storage
+ls -alF /storage/annotations
+unset CUDA_VISIBLE_DEVICES
+
+PYCMD=$(cat <<EOF
+import pycuda
+from pycuda import compiler
+import pycuda.driver as drv
+
+drv.init()
+print("%d device(s) found." % drv.Device.count())
+           
+for ordinal in range(drv.Device.count()):
+    dev = drv.Device(ordinal)
+    print (ordinal, dev.name())
+EOF
+)
+
+python -c "$PYCMD"
 
-exec python -m torch.distributed.launch --nproc_per_node=1 \
-        main.py --batch-size 32 \
-          --mode benchmark-training \
-          --benchmark-warmup 100 \
-          --benchmark-iterations 100 \
-          --data /storage
+export NCCL_DEBUG=INFO
+export CUDA_VISIBLE_DEVICES=0
+exec python -u -m bind_launch --nsockets_per_node=1 --ncores_per_socket=1 --nproc_per_node=1 \
+     train.py --epochs 2 \
+              --warmup-factor 0 \
+              --threshold=0.25 \
```

Comment:
can you make the number of `epochs` and `threshold` configurable from the entrypoint?
by default, let it empty, and in the script, if empty, use these defaults values:
```
+ THRESHOLD=0.23
+ NUM_EPOCHS=80
```
> --threshold=0.25 
I think threshold `0.25` is unreachable (eg, asymptotical to it), `0.23` is the sensible limit ;-)